package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "ghost-gql",
-  "version": "0.0.4",
+  "version": "0.0.5",
   "description": "Filter query language for Ghost",
   "main": "index.js",
   "scripts": {
-    "lint": "jshint lib/*.js test/*.js && jscs lib/*.js test/*.js",
-    "test": "npm run build && npm run lint && mocha",
-    "build": "jison src/gql.y src/gql.l -o dist/parser.js",
-    "coverage": "istanbul cover -x src --dir=test/coverage --report=lcov ./node_modules/mocha/bin/_mocha -- test/*_spec.js",
+    "lint": "jshint src/*.js test/*.js && jscs src/*.js test/*.js",
+    "test": "npm run lint && mocha",
```

Comment:
I just checked in the parser instead of building it each time. I don't expect it to change often.